audbackend/core/backend.py

```diff
@@ -308,34 +308,45 @@ def latest_version(
 
     def _ls(
             self,
-            folder: str,
+            path: str,
     ) -> typing.List[typing.Tuple[str, str, str]]:  # pragma: no cover
-        r"""List all files under folder.
+        r"""List all files under (sub-)path.
 
-        If folder does not exist an error should be raised.
+        * If path does not exist an error should be raised.
+        * If path ends on `/` it is a sub-path
 
         """
         raise NotImplementedError()
 
     def ls(
             self,
-            folder: str = '/',
+            path: str = '/',
             *,
             latest_version: bool = False,
             pattern: str = None,
             suppress_backend_errors: bool = False,
     ) -> typing.List[typing.Tuple[str, str]]:
-        r"""List all files under folder.
+        r"""List files on backend.
 
         Returns a sorted list of tuples
         with path and version.
-        When ``folder`` is set to the
-        root of the backend (``'/'``)
+        If a full path
+        (e.g. ``/sub/file.ext``)
```

Comment:
Maybe change to 

```suggestion
        (e.g. ``sub/file.ext``)
```
as ``/`` is not really needed.

Reply:
You are right, at the moment it's not needed. So I will apply your suggestion for now.

However, I think we should enforce that a backend path has to start with `/`. Maybe someday there will be a use-case for using relative backend paths, so it's better to now insist on absolute paths. But I will tackle this once when we solve https://github.com/audeering/audbackend/issues/57.